src/common.js

```diff
@@ -0,0 +1,7 @@
+function ignorePipeErrors(proc) {
```

Comment:
I can see the reasoning on why it's in a separate file, but in that case let's call it `ignore-pipe-errors.js` :)